app/jobs/application_job.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationJob < ActiveJob::Base
```

Comment:
Whats this file about?

Reply:
This is our base class for all jobs. This will be inherited by all jobs. This is where you can define common methods that will be helpful across all of your jobs. 

Rails by default comes with an asynchronous queuing implementation that runs jobs in the backend. These jobs can be everything from regularly scheduled clean-ups, to billing charges, to mailings.

Reply:
What is a `job` ?

Reply:
Jobs can be any Ruby class or module that does something specified by the code when it responds to triggers we set.